extensions/amp-story/amp-story.md

```diff
@@ -338,6 +338,18 @@ A URL to the [story poster](#posters) in square format (1x1 aspect ratio).
 
 A URL to the [story poster](#posters) in landscape format (4x3 aspect ratio).
 
+##### live-story [optional]
+
+Enables the [Live story](#Live-story) functionality. Read more on that section.
+
+##### live-story-disabled [optional]
+
+Disables the [Live story](#Live-story) functionality. Read more on that section.
+
+##### data-poll-interval [optional]
+
+To be used together with the `live-story` attribute. Time (in milliseconds) interval between checks for new content (15000 ms minimum is enforced). If no `data-poll-interval` is provided it will default to the 15000 ms minimum.
```

Comment:
Reword as:

Used with the `live-story` attribute. Time interval (in milliseconds) between checks for new content. If no `data-poll-interval` is provided it with default to the 15000 millisecond minimum. A value under 15000 milliseconds is invalid.